client/CMusicHandler.h

```diff
@@ -83,7 +84,7 @@ class CSoundHandler: public CAudioBase
 	// Sets
 	std::vector<soundBase::soundID> pickupSounds;
 	std::vector<soundBase::soundID> battleIntroSounds;
-	std::map<Terrain, soundBase::soundID> horseSounds;
+	std::map<TTerrainId, soundBase::soundID> horseSounds;

```

Comment:
T stands for Type

Reply:
As far as I remember we abandoned the idea to prefix class names with C T etc. And this was agreed with @alexvins or @ArseniyShestakov. I only agree with using T as a marker of that it is a template type. Like TVector. Same for variables. Also I see that quite a lot of places prefer ID uppercased so we have a lot of them with Id as well)

Reply:
Let's keep camel case in coding guidelines. If Type prefix is needed, let's use TypeMyObject, no TMyObject. But TerrainId for me looks better rather than TypeTerrainId or TTerrainId

Reply:
It's not like I invented something new:

![image](https://user-images.githubusercontent.com/2566990/192746238-2e1d257f-1ed9-4d8c-bb74-d29893275614.png)

Reply:
Yes there is a lot of TSomething and CSomething but it was agreed that we no longer need at least C prefix. If TTerrainId is not typedef there should be no T. In case of typedef it is discussable. typedef looks like new type definition so we can treat it as a regular class or whatever. So the question is if we need to know that this is typedef exactly or not? In most cases I expect the answer to be no. Also further refactoring will be easier if we decide to convert typedef to class/struct/enum whatever

Reply:
If I just remove T prefix, will that be OK?

Reply:
Okay, I changed it. Looks like the final commit for this MR.